hooks/publish.py

```diff
@@ -306,22 +306,8 @@ def validate(self, task_settings, item):
 
         attr_list = ("publish_type", "publish_name", "publish_version", "publish_path",
                      "publish_symlink_path", "publish_linked_entity_name")
-        for attr in attr_list:
-            try:
-                method = getattr(self, "_get_%s" % attr)
-                item.properties[attr] = method(item, task_settings)
-            except Exception:
-                self.logger.error(
-                    "Unable to determine '%s' for item: %s" % (attr, item.name),
-                    extra={
-                        "action_show_more_info": {
-                            "label": "Show Error Log",
-                            "tooltip": "Show the error log",
-                            "text": traceback.format_exc()
-                        }
-                    }
-                )
-                return False
+        if not self._validate_getter_attrs(item, task_settings, attr_list):
```

Comment:
Do me a favor and switch the ordering of the `item` and `task_settings` parameters on this signature and all subsequent signatures. I was intending on doing this anyways to match the standardized signatures for other public facing methods.

Reply:
Sure!